base_report_to_printer/report.py

```diff
@@ -44,11 +44,12 @@ def print_document(self, cr, uid, ids, report_name, html=None,
             )
         return printer.print_document(report, document, report.report_type)
 
-    @api.v8
+    @api.v8  # noqa
```

Comment:
Otherwise I get an flake8 error because of repetition of `print_document` and `get_pdf`